lib/auth/grpcserver.go

```diff
@@ -2229,6 +2229,8 @@ func (g *GRPCServer) authenticate(ctx context.Context) (*grpcContext, error) {
 		// between connection failed and access denied
 		if trace.IsConnectionProblem(err) {
 			return nil, trace.ConnectionProblem(err, "[10] failed to connect to the database")
+		} else if trace.IsNotFound(err) {
+			return nil, trace.Wrap(err, "[10] access denied")
```

Comment:
will this show up as `[10] access denied` on the client side, or will the original error be included?

Reply:
The original error is included. It looked something like `user "teleport" is not found`.

Reply:
Cool, please leave a comment here to mention it.
It's easy to assume that the error is masked (the `trace` package is pretty inconsistent about it).